std_msgs/Int64 id_poubelle
std_msgs/Int64 state
---
std_msgs/Bool res
